### ISSUE TITLE
DataTable SearchBar: An interactive element/widget must have an accessible name

### ISSUE THREAD

**Issue by megbridge** [carbon-design-system/carbon#3036]
In a DataTable with a SearchBar, when running DAP, the following violation occurs:
An interactive element/widget must have an accessible name.

## Environment

> Operating system

> Browser
Chrome

> Automated testing tool and ruleset
DAP IBM Accessibility August 2018

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
V10.2

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?
An interactive element/widget must have an accessible name.

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
https://codesandbox.io/embed/datatable-pld42
